binder/postBuild

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+import os
+from mne.datasets import eegbci
+from mne_bids.datasets import fetch_brainvision_testing_data, fetch_faces_data
+
+# Make a directory to save the data to
+home = os.path.expanduser('~')
+mne_dir = os.path.join(home, 'mne_data')
+if not os.path.exists(mne_dir):
+    os.makedirs(mne_dir)
+
+########## Fetch data #########################################################
+print("Downloading data for examples...")
+
+# EEG BCI data
+# Define which tasks we want to download.
+tasks = [2,  # This is 2 minutes of eyes closed rest
+         4,  # This is run #1 of imagining to close left or right fist
+         12]  # This is run #2 of imagining to close left or right fist
+
+# Download the data for subjects 1 and 2
+for subj_idx in [1, 2]:
+    eegbci.load_data(subject=subj_idx,
+                     runs=tasks,
+                     path=mne_dir,
+                     update_path=True)
+
+# Faces data
+subject_ids = [1]
+runs = range(1, 7)
+
+data_path = os.path.join(os.path.expanduser('~'), 'mne_data')
+repo = 'ds000117'
+fetch_faces_data(data_path, repo, subject_ids)
+
+# Sample data
+from mne.datasets import sample
+
+data_path = sample.data_path()
+
+# Brainvision
+examples_dir = fetch_brainvision_testing_data()
```

Comment:
isn't this going to really slow down the loading of binder? I'd rather prefer the users do it after opening the notebook, so they can at least see some progressbars. What do you think @choldgraf ?

Reply:
that's because I don't believe that a version has been pushed to Binder that does the pre-loading of data yet...in the end, the environment the user gets should already have `~/mne_data` populated with the relevant datasets so the data_path function just returns the string w/o downloading

Reply:
okay I see, what needs to be done to push the version to Binder? are you saying that eventually the link above for the binder will open quasi-instantaneously instead of downloading the data for every instance requested by the user?

Reply:
yes - once the data is baked into the docker image, it'll load much faster (though the image itself will be quite large since the data that's currently downloaded amounts to tens of gigabytes to run all examples)

Reply:
hmm ... I see. My only constraint is that the binder should load (almost) instantaneously (this is not the case currently). As long as that happens, I'm fine :)

Reply:
well it'll be much faster if we don't deal in tens of gigabytes of data :-)

Reply:
okay let me try to play around with a fork before I comment further :-P